docs/pages/application-access/introduction.mdx

```diff
@@ -37,53 +36,45 @@ internal dashboards and applications, such as:
 
 ## Getting started
 
-Get started with Application Access in a 10 minute [guide](./getting-started.mdx).
+Learn how to register an application to Teleport with Get started with in our
+[getting started guide](./getting-started.mdx).
+
+## Protect cloud provider APIs
+
+You can use Teleport to provide secure access to your cloud provider's APIs.
+This means that you can prevent unauthorized usage of management consoles and
+CLI tools with the same RBAC system you use to protect your infrastructure.
+
+- [AWS Console and CLI Applications](./clud-apis/aws-console.mdx): How to access
+  AWS Management Console, AWS CLI, and AWS SDKs with Teleport.
+- [Google Cloud CLI Applications](./cloud-apis/google-cloud.mdx): How to access
+  Google Cloud CLI applications and SDKs with Teleport.
+- [Azure CLI Applications](./cloud-apis/azure.mdx): How to access Azure CLI
+  applications and SDKs with Teleport.
 
-## Access guides
+## Protect internal applications
 
-These guides explain basic Teleport Application Access usage.
+You can use Teleport to enable secure access to internal applications. For
+example, a load balancer might display network telemetry through a control panel
+but might lack the ability to authenticate with and be accessed by resources
+outside your private network.
+
+Teleport lets team members access these resources securely, even outside a
+private network, with no shared secrets.
+
+These guides explain how to protect internal applications  with Teleport:
 
 - [Web App Access](./guides/connecting-apps.mdx): How to access web apps with Teleport Application Access.
 - [TCP App Access (Preview)](./guides/tcp.mdx): How to access plain TCP apps with Teleport Application Access.
 - [API Access](./guides/api-access.mdx): How to access REST APIs with Teleport Application Access.
-- [AWS Access](./guides/aws-console.mdx): How to access AWS Management Console, AWS CLI, and AWS SDKs with Teleport Application Access.
 - [Dynamic Registration](./guides/dynamic-registration.mdx): Register/unregister apps without restarting Teleport.
 - [Interactive Lab](https://play.instruqt.com/teleport/invite/rgvuva4gzkon): Try Teleport using our guided Teleport Application Access lab.
 
-## JWT guides
+## Use Teleport-signed JSON web tokens
```

Comment:
Should JSON Web Tokens be capitalized? It's a proper noun, isn't it? jwt.io seems to capitalize it